File: Go/217_Contains_Duplicate.go
func containsDuplicate(nums []int) bool {
    freqmap:=make(map[int]int)
    for i:=0;i<len(nums);i++{
        temp:=nums[i]
        if freqmap[temp]>0{
            return true
        }
        freqmap[temp]++
    }
    return false
}

File: Java/217_Contains_Duplicate.java
class Solution {
    public boolean containsDuplicate(int[] nums) {
        Set<Integer> res=new HashSet<Integer>();
        for(int n : nums){
            if(res.contains(n)){
                return true;
            }
            res.add(n);
        }
        return false;
    }        
}